src/styles/user-profile.less

```diff
@@ -0,0 +1,246 @@
+            }
+
+            &:hover {
+                background-color: lighten(@bc-primary-btn-bg, 10%);
+            }
+
+            &:active {
+                background-color: @bc-primary-btn-bg-down;
+                box-shadow: inset 0 1px 0 @bc-shadow-small;
+            }
+        }
+
+        &.btn-default {
+            background-color: @bc-btn-bg;
+            border: 1px solid @bc-btn-border;
+            box-shadow: inset 0 1px 0 @bc-highlight;
+
+            &:hover {
+                background-color: lighten(@bc-btn-bg, 10%);
+            }
+
+            &:active {
+                background-color: @bc-btn-bg-down;
+                box-shadow: inset 0 1px 0 @bc-shadow-small;
+            }
+        }
+
+        &.btn-link {
+            background: none;
+            border: none;
+            box-shadow: none;
+            color: @bc-text-thin-quiet;
+
+            i {
+                margin-right: 5px;
+            }
+
+            &:hover {
+                color: @bc-text;
+                text-decoration: none;
+            }
+        }
+    }
+}
+
+.dark .profile-popup {
+    background-color: @dark-bc-menu-bg;
+    color: @dark-bc-menu-text;
+    box-shadow: 0 3px 9px @dark-bc-shadow;
+
+    .popup-header {
+        border-bottom: 1px solid @dark-bc-menu-separator;
+    }
+
+    .popup-title {
+        color: @dark-bc-menu-text;
+    }
+
+    .user-avatar {
+        background-color: @dark-bc-primary-btn-bg;
+        color: @dark-bc-text-alt;
+    }
+
+    .user-name {
+        color: @dark-bc-menu-text;
+    }
+
+    .progress-bar {
+        background-color: @dark-bc-input-bg;
+
+        .progress-fill {
+            background-color: @dark-bc-primary-btn-bg;
+        }
+    }
+
+    .btn {
+        &.btn-primary {
+            background-color: @dark-bc-primary-btn-bg;
+            border: 1px solid @dark-bc-primary-btn-border;
+            box-shadow: inset 0 1px 0 @dark-bc-highlight;
+            color: @dark-bc-text-alt;
+
+            &:hover {
+                background-color: lighten(@dark-bc-primary-btn-bg, 10%);
+            }
+
+            &:active {
+                background-color: @dark-bc-primary-btn-bg-down;
+                box-shadow: inset 0 1px 0 @dark-bc-shadow-small;
+            }
+        }
+
+        &.btn-default {
+            background-color: @dark-bc-btn-bg;
+            border: 1px solid @dark-bc-btn-border;
+            box-shadow: inset 0 1px 0 @dark-bc-highlight;
+            color: @dark-bc-text;
+
+            &:hover {
+                background-color: lighten(@dark-bc-btn-bg, 10%);
+            }
+
+            &:active {
+                background-color: @dark-bc-btn-bg-down;
+                box-shadow: inset 0 1px 0 @dark-bc-shadow-small;
+            }
+        }
+
+        &.btn-link {
+            color: @dark-bc-text-thin-quiet;
+
+            &:hover {
+                color: @dark-bc-text;
+            }
+        }
+    }
+}
+
+@keyframes dropdown {
```

Comment:
dropdown var name is too generic and may cause collisions in future. use user-profile-dropdown-animation or something like that.